install/ansible/env.json

```diff
@@ -1,20 +1,20 @@
-{ 
-  "docker_version":"__DOCKER_VERSION__", 
-  "aci_gw_image":"contiv/aci-gw:__ACI_GW_VERSION__",
-  "contiv_network_version":"__CONTIV_VERSION__",
-  "env": "{}",
+{
+  "docker_version": "__DOCKER_VERSION__",
+  "aci_gw_image": "contiv/aci-gw:__ACI_GW_VERSION__",
+  "contiv_network_version": "__CONTIV_VERSION__",
+  "env": {},
   "etcd_peers_group": "netplugin-master",
   "service_vip": "__NETMASTER_IP__",
-  "validate_certs": "no",
+  "validate_certs": false,
   "cluster_store": "__CLUSTER_STORE__",
   "auth_proxy_image": "contiv/auth_proxy:__API_PROXY_VERSION__",
-  "docker_reset_container_state": "False",
-  "docker_reset_image_state": "False",
-  "etcd_cleanup_state": "False",
-  "auth_proxy_local_install": "False",
-  "contiv_network_local_install": "False",
-  "vxlan_port": "4789",
+  "docker_reset_container_state": __DOCKER_RESET_CONTAINER_STATE__,
+  "docker_reset_image_state": __DOCKER_RESET_IMAGE_STATE__,
+  "etcd_cleanup_state": __ETCD_CLEANUP_STATE__,
```

Comment:
these three I think are supposed to be customer controlled, I think should not arbitrarily overwritten during install/uninstall . . .

Reply:
nvm, it won't match if customer edited